src/usr/local/buildpack/tools/npm.sh

```diff
@@ -10,9 +10,13 @@ fi
 
 npm install -g npm@${TOOL_VERSION}
 
+hash -d npm
```

Comment:
This helps the subsequent `npm --version` to succeed although doesn't help the caller of `install-tool`.

Reply:
yes, it's because of https://github.com/containerbase/buildpack/blob/95725c1305a116cb654ff3c6d6b5562c1cfeb111/src/usr/local/buildpack/tools/node.sh#L48

So npm is already on path.

Maybe we should always add the npm shim, so it's on path before all others, so we don't need the rehash?

Reply:
Should that be in this PR, or another? In the meantime we can keep using hash in Renovate

Reply:
Actually let's do separate.. this PR doesn't make anything worse